tools/python/maps_generator/update_generation_order.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+from multiprocessing.pool import ThreadPool
+from typing import Tuple
+
+from maps_generator.checks.logs import logs_reader
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description="This script generates file with countries that are ordered by time."
```

Comment:
s/by time/by time needed to generate them/

Reply:
++